foundation/foundation-configuration-store/src/main/java/com/boozallen/aissemble/configuration/store/ConfigLoader.java

```diff
@@ -167,8 +170,14 @@ private Set<Property> reconcileConfigs(Set<Property> baseConfigs, Set<Property>
      * @param properties to be written to store
      */
     public void write(Set<Property> properties) {
-        propertyDao.write(properties);
-        logger.info("Successfully wrote all properties to the store.");
+        try {
+                propertyDao.write(properties);
+                logger.info("Successfully wrote all properties to the store.");
+                updateLoadStatus(fullyLoaded);
```

Comment:
If it successfully wrote, could we just do `updateLoadStatus(true);`?

Reply:
yes I've changed it to just updateLoadStatus(true)